packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -97,9 +98,11 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         if (is_positions_drawer_on) {
             togglePositionsDrawer(); // TODO: hide drawer inside logout, once it is a mobx action
         }
+
+        setShouldShowGlobalLoader(true);
```

Comment:
also why we need loader here @sergei-deriv 🙏

Reply:
@mitra-deriv tried to answer above